src/SFA.DAS.ApplyService.Application/Apply/Assessor/CreateAssessorPageReviewOutcomesHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Assessor
+{
+    public class CreateAssessorPageReviewOutcomesHandler : IRequestHandler<CreateAssessorPageReviewOutcomesRequest>
+    {
+        private readonly IAssessorRepository _repository;
+        private readonly ILogger<CreateAssessorPageReviewOutcomesHandler> _logger;
+
+        public CreateAssessorPageReviewOutcomesHandler(IAssessorRepository repository, ILogger<CreateAssessorPageReviewOutcomesHandler> logger)
+        {
+            _repository = repository;
+            _logger = logger;
+        }
+
+        public async Task<Unit> Handle(CreateAssessorPageReviewOutcomesRequest request, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation($"CreateAssessorPageReviewOutcomes for ApplicationId '{request.AssessorPageReviewOutcomes.First().ApplicationId}'");
+
+            foreach (var outcome in request.AssessorPageReviewOutcomes)
+            {
+                await _repository.SubmitAssessorPageOutcome(outcome.ApplicationId,
```

Comment:
I'd be tempted to write a new method in the repository that can bulk insert all of these in one go.

Reply:
Changed to bulk insert